ehitchhiking-ui/src/app/shared/components/menu/menu.component.ts

```diff
@@ -35,14 +35,6 @@ export class MenuComponent implements OnInit {
 		});
 	}
 
-	openRatePassengersDialog(): void {
```

Comment:
Pls remove importing of RatePassengersModule in the MenuModule and add this import in the module responsible for the "History" modal

Reply:
fixed